{"name": "test_login_non_exist_courier_failed[login]", "status": "passed", "parameters": [{"name": "key", "value": "'login'"}], "start": 1726088787075, "stop": 1726088787566, "uuid": "8a4ecd6f-efc7-43e1-b63e-33cbaa5cf44e", "historyId": "d8eee50437a6935fd94d31780c896374", "testCaseId": "39bf59179781146867adebade5da90ff", "fullName": "tests.test_login_courier_endpoint.TestLoginCourier#test_login_non_exist_courier_failed", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login_courier_endpoint"}, {"name": "subSuite", "value": "TestLoginCourier"}, {"name": "host", "value": "HOME"}, {"name": "thread", "value": "42220-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login_courier_endpoint"}]}